extension/js/common/composer.ts

```diff
@@ -414,22 +416,19 @@ export class Composer {
     await this.composerContacts.setEmailsPreview(this.getRecipients());
     await this.renderComposeTable();
     if (this.canReadEmails) {
-      const determined = await this.app.emailProviderDetermineReplyMsgHeaderVariables();
-      if (determined) {
-        this.urlParams.subject = `${(method === 'reply' ? 'Re' : 'Fwd')}: ${this.urlParams.subject}`;
-        this.composerSendBtn.additionalMsgHeaders['In-Reply-To'] = determined.headers['In-Reply-To'];
-        this.composerSendBtn.additionalMsgHeaders.References = determined.headers.References;
-        if (!this.urlParams.draftId) { // if there is a draft, don't attempt to pull quoted content. It's assumed to be already present in the draft
-          (async () => { // not awaited because can take a long time & blocks rendering
-            const footer = this.getFooter();
-            await this.composerQuote.addTripleDotQuoteExpandBtn(determined.lastMsgId, method, footer);
-            if (this.composerQuote.messageToReplyOrForward && this.composerQuote.messageToReplyOrForward.isSigned) {
+      this.urlParams.subject = `${(method === 'reply' ? 'Re' : 'Fwd')}: ${this.urlParams.subject}`;
+      if (!this.urlParams.draftId) { // if there is a draft, don't attempt to pull quoted content. It's assumed to be already present in the draft
+        (async () => { // not awaited because can take a long time & blocks rendering
+          const footer = this.getFooter();
+          await this.composerQuote.addTripleDotQuoteExpandBtn(this.urlParams.replyMsgId, method, footer);
+          if (this.composerQuote.messageToReplyOrForward) {
+            this.composerSendBtn.additionalMsgHeaders['In-Reply-To'] = this.composerQuote.messageToReplyOrForward.headers['in-reply-to'];
+            this.composerSendBtn.additionalMsgHeaders.References = this.composerQuote.messageToReplyOrForward.headers.references;
```

Comment:
Here also a mistake, you can compare with previous implementation which does 
```js
'References': threadMsgRefsLast + ' ' + replyMsgIdLast
```
When unsure, you can check the spec: https://tools.ietf.org/html/rfc4021#section-2.1.10

> The message identifier(s) of other message(s) to which the current
      message may be related.  In RFC 2822, the definition was changed
      to say that this header field contains a list of all Message-IDs
      of messages in the preceding reply chain.  Defined as standard by
      RFC 822.